packages/website/src/components/NavItem.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'gatsby'
+import { Button, Text } from '@containous/faency'
+
+function NavItem({ children, isExternal, active, ...props }) {
+  return (
+    <Button
+      as={Link}
+      {...props}
+      variant={active ? 'active' : undefined}
+      px={5}
+      minHeight={6}
+      style={{ justifyContent: 'flex-start' }}
+    >
+      <Text size={2} textColor={active ? 'blue' : undefined} mr={isExternal ? 1 : 0}>
+        {children}
+      </Text>
+      {/* {isExternal && <ExternalLinkIcon />} */}
```

Comment:
Do we have plans to use this `isExternal` in the near future? Otherwise, I think it's no longer relevant [here](https://github.com/containous/faency/pull/60/files#diff-db7d1201bf96f805cde5a01ad2c71b38R124) and [here](https://github.com/containous/faency/pull/60/files#diff-db7d1201bf96f805cde5a01ad2c71b38R127).